backend/protocol/api/_mcp.py

```diff
@@ -29,6 +34,58 @@
 )
 from protocol.api._services import models_service
 
+
+class MCPModel(BaseModel):
```

Comment:
not the right file for this definition. Please follow the existing structure, there is no model definition or logic in this file.